src/mainboard/raptor-cs/talos-2/board.fmd

```diff
@@ -11,7 +11,7 @@
 
 FLASH@0 CONFIG_ROM_SIZE {
 	BIOS@0 CONFIG_ROM_SIZE {
-#if !CONFIG_BOOTBLOCK_IN_SEEPROM
+#if !CONFIG(BOOTBLOCK_IN_SEEPROM)
```

Comment:
@krystian-hebel I guess i tried to make the linter happy and made the build fail at the same time: 
https://github.com/3mdeb/coreboot/runs/2719266434?check_suite_focus=true#step:4:327

Reply:
I can see no other .fmd files in tree using any kind of CONFIG dynamically. And the linter checks for the syntax it expects to find in the regular source files

Reply:
I can see that in case of different platforms rather a set of different .fmd files is provided and the proper one is used. Maybe we should do that as well?

Reply:
I we have to... Then in Kconfig we should add different defaults depending on `BOOTBLOCK_IN_SEEPROM` as well.

Reply:
@krystian-hebel Please take a look if it makes sense: https://github.com/3mdeb/coreboot/pull/65/commits/8a2c5569333f608594d3c94f1d280608f5957364

Reply:
It is this one after rebasing: https://github.com/3mdeb/coreboot/pull/65/commits/bccd98ed5b500338b3954331db7ba13e322f10e1

Both variants (bootblock in PNOR/EEPROM) are already passing in the CI job.